mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -240,6 +240,11 @@ mesheryctl perf apply local-perf --url https://192.168.1.15/productpage --mesh i
 		log.Info("Initiating Performance test ...")
 
 		resp, err := client.Do(req)
+
+		// failsafe for having an expired token
```

Comment:
We could add a comment here mentioning that expired tokens results in a redirect and it will keep redirecting since we are in a CLI client. So, we can check for the response code for redirect to check if the token has expired.

This comment could be helpful for future developers.